management/clipper_manager.py

```diff
@@ -270,9 +266,13 @@ def register_application(self,
         default_output : float
             The default prediction to use if the model does not return a prediction
             by the end of the latency objective.
-        slo_micros : int, optional
+        slo_micros : int
             The query latency objective for the application in microseconds.
-            Default is 20,000 (20 ms).
+            If Clipper cannot process a query within the latency objective,
+            the default output is returned. Therefore, it is recommended that
+            the objective not be set aggressively low unless absolutely necessary.
+            Starting at 40000 (40 ms) is a good for testing, but the optimal
+            the optimal latnecy objective will vary depending on the application.
```

Comment:
Latency is mis-spelled on the last line.

Can you add a note indicating that this is the processing latency between Clipper receiving a request and sending a response. It does not account for network latencies before a request is received or after a response is sent.